mongodbforms/fieldgenerator.py

```diff
@@ -115,7 +115,7 @@ def boolean_field(self, value):
         return value.lower() == 'true'
 
     def get_field_label(self, field):
-        if getattr(field, 'verbose_name', None):
+        if hasattr(field, 'verbose_name') and field.verbose_name:
```

Comment:
This doesn't strike me as very pythonic. Why make this change? And if there's a specific reason, I think you need to add a comment to avoid someone optimizing back to `getattr(…, default=None)` at a later date

Reply:
I don't believe I've meant to make that change in this PR... there must be something wrong in my git tree.

Reply:
Nevermind, that must have been an old local change that I hadn't cleaned up for #24. Rewriting the commit to get rid of it. 7587a40 :white_check_mark: